src/settings/components/ExternalDataSourcesConfig/ExternalDataSourceForm/ExternalDataSourceForm.test.js

```diff
@@ -1,40 +1,25 @@
-import { Field } from 'react-final-form';
-
 import { waitFor } from '@folio/jest-config-stripes/testing-library/react';
-
-import {
-  Button,
-  Checkbox,
-  renderWithIntl,
-  Select,
-  TestForm,
-  TextArea,
-  TextField,
-} from '@folio/stripes-erm-testing';
+import { TestForm, renderWithIntl, Checkbox, Select, TextArea, TextField } from '@folio/stripes-erm-testing';
 
 import translationsProperties from '../../../../../test/helpers';
-
-import ExternalDataSourcesEdit from './ExternalDataSourcesEdit';
+import ExternalDataSourceForm from './ExternalDataSourceForm';
 
 const onCancelMock = jest.fn();
 const onSaveMock = jest.fn();
 const onSubmit = jest.fn();
 
 const initialValues = {
   'externalKbs': [{
```

Comment:
Could JS objects not have string keys wherever possible please? So instead of `{'externalKbs': []}` it's `{externalKbs: []}` etc

Reply:
This applies to _all_ test resources. It'd just be nice to have them consistent